### PR TITLE
Initial draft of security reporting processs

### PR DESCRIPTION
**Pull request by lmbr-pip**
Deliver docs content for: https://github.com/o3de/o3de/issues/6803

Signed-off-by: Pip Potter <61438964+lmbr-pip@users.noreply.github.com>

Screen Shot 2022-03-04 at 12 07 32 PM (2)

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/contributing/security/_index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+weight: 100
+---
+
+## Open 3D Engine: Security
+This page describes O3DE's Security and Disclosure Policies. 
+
+O3DE values all contributions that help keep O3DE secure. [SIG-Security](https://github.com/o3de/sig-security) volunteers will thoroughly research and investigate all reported vulnerabilities. 
+
+# Report a Vulnerability
+We encourage all vulnerabilities to be reported through the security@o3de.org list. Vulnerability reports should follow the same information reporting format as [standard O3DE issues](https://github.com/o3de/o3de/blob/development/.github/ISSUE_TEMPLATE/bug_template.md) along with any security specific details.
```
Comment (myfoostrong) [inline content/docs/contributing/security/_index.md:14]: security@o3de.org `mailing` list.

File: content/docs/contributing/security/_index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+* You have discovered or are aware of a security issue relating to an O3DE dependency that may impact O3DE.
+     * If the issues is in a product that O3DE depends upon, we strongly encourage you to engage with that project's vulnerability reporting mechanisms. 
+
+## When Should I NOT Report a Vulnerability?
+* You need help setting up or have issues relating to the use of security dependencies such as OpenSSL or TLS/DTLS 
+* You need help applying a security related fix to your O3DE applications. 
+* The issue is not related to security. Please report issue using the normal [O3DE issue reporting process](https://github.com/o3de/o3de/issues).
+
+## Can I Report a Vulnerability via GitHub Issues?
+If there is public disclosed security information, such as a report in the [National Vulnerability Database](https://nvd.nist.gov/) then GitHub issues can be used to communicate security vulnerabilities impacting O3DE.
```
Comment (myfoostrong) [inline content/docs/contributing/security/_index.md:27]: `publicly` disclosed security information

File: content/docs/contributing/security/_index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+# Security Vulnerability Response
+When a security vulnerability is reported via security@o3de.org, a member of the SIG-Security response team should respond within 3 working days. Once the issue has been acknowledged, the SIG-Security response team will follow its [documented response process](https://github.com/o3de/sig-security) to ensure the reporter is updated and informed of current actions and next steps.
+
+Any vulnerability information will treated as confidential and will be not shared outside of the security response process. Vulnerability information will not be publicly shared or disclosed without the consent of the reporter.
+
+# Public Disclosure
+The public disclosure date of any security vulnerability will be negotiated between SIG-Security and the vulnerability reporter. The SIG aims to publicly disclose the issue as soon as possible, ideally once a mitigation as been identified. Disclosure time can range from almost immediately, for publicly known issues, to a few weeks depending on the complexity of issue and coordination required.
```
Comment (myfoostrong) [inline content/docs/contributing/security/_index.md:35]: ideally once a mitigation `has` been identified

**Review by myfoostrong — CHANGES_REQUESTED** [review]

File: content/docs/contributing/security/_index.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+O3DE values all contributions that help keep O3DE secure. [SIG-Security](https://github.com/o3de/sig-security) volunteers will thoroughly research and investigate all reported vulnerabilities. 
+
+# Report a Vulnerability
+We encourage all vulnerabilities to be reported through the security@o3de.org list. Vulnerability reports should follow the same information reporting format as [standard O3DE issues](https://github.com/o3de/o3de/blob/development/.github/ISSUE_TEMPLATE/bug_template.md) along with any security specific details.
+
+## When Should I Report a Vulnerability?
+* You have discovered a potential security issue relating to O3DE. 
+* You have discovered or are aware of a security issue relating to an O3DE dependency that may impact O3DE.
+     * If the issues is in a product that O3DE depends upon, we strongly encourage you to engage with that project's vulnerability reporting mechanisms.
```
Comment (allisaurus) [inline content/docs/contributing/security/_index.md:19]: should we say "...encourage you to **also** engage with that project's vulnerability reporting mechanisms"? (I'm assuming we'd still want the report)

File: content/docs/contributing/security/_index.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+## When Should I NOT Report a Vulnerability?
+* You need help setting up or have issues relating to the use of security dependencies such as OpenSSL or TLS/DTLS 
+* You need help applying a security related fix to your O3DE applications. 
+* The issue is not related to security. Please report issue using the normal [O3DE issue reporting process](https://github.com/o3de/o3de/issues).
+
+## Can I Report a Vulnerability via GitHub Issues?
+If there is public disclosed security information, such as a report in the [National Vulnerability Database](https://nvd.nist.gov/) then GitHub issues can be used to communicate security vulnerabilities impacting O3DE.
+
+# Security Vulnerability Response
+When a security vulnerability is reported via security@o3de.org, a member of the SIG-Security response team should respond within 3 working days. Once the issue has been acknowledged, the SIG-Security response team will follow its [documented response process](https://github.com/o3de/sig-security) to ensure the reporter is updated and informed of current actions and next steps.
```
Comment (allisaurus) [inline content/docs/contributing/security/_index.md:30]: I feel like we should make the "**documented response process**" link go somewhere real before publishing this (even the in-flight response process RFC), unless we plan to add a very visible "RESPONSE PROCESS" link to the `sig-security` README/landing page in the near future

File: content/docs/contributing/security/_index.md (hunk 1 above)
Comment (lmbr-pip) [reply content/docs/contributing/security/_index.md:14]: Fixed.

File: content/docs/contributing/security/_index.md (hunk 2 above)
Comment (lmbr-pip) [reply content/docs/contributing/security/_index.md:27]: Fixed.

File: content/docs/contributing/security/_index.md (hunk 4 above)
Comment (lmbr-pip) [reply content/docs/contributing/security/_index.md:19]: Yes. Good catch. Fixed.

File: content/docs/contributing/security/_index.md (hunk 5 above)
Comment (lmbr-pip) [reply content/docs/contributing/security/_index.md:30]: Theres nothing good to link to so am erring on the side that we will fix this up immediately on publication of the RESPONSE_PROCESS.md or similar (once the RFC is accepted - I could link to the RFC I suppose). For now response is adhoc and owned by SIG-Security.

File: content/docs/contributing/security/_index.md (hunk 3 above)
Comment (lmbr-pip) [reply content/docs/contributing/security/_index.md]: Fixed.

**Review by allisaurus — APPROVED** [review]

File: content/docs/contributing/security/_index.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+---
+linktitle: Security
+title: "Contributing to O3DE: Security"
+toc: true
+weight: 100
+---
+
+## Open 3D Engine: Security
+This page describes O3DE's Security and Disclosure Policies.
```
Comment (sptramer) [inline content/docs/contributing/security/_index.md:9]:
```suggestion
This page describes the Security and Disclosure policies for **Open 3D Engine (O3DE)**. 
```

- Style guide application: No possessive on project name, formatting

File: content/docs/contributing/security/_index.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+weight: 100
+---
+
+## Open 3D Engine: Security
+This page describes O3DE's Security and Disclosure Policies. 
+
+O3DE values all contributions that help keep O3DE secure. [SIG-Security](https://github.com/o3de/sig-security) volunteers will thoroughly research and investigate all reported vulnerabilities. 
+
+# Report a Vulnerability
+We encourage all vulnerabilities to be reported through the security@o3de.org mailing list. Vulnerability reports should follow the same information reporting format as [standard O3DE issues](https://github.com/o3de/o3de/blob/development/.github/ISSUE_TEMPLATE/bug_template.md) along with any security specific details.
```
Comment (sptramer) [inline content/docs/contributing/security/_index.md:14]:
```suggestion
We encourage all vulnerabilities to be reported through the [security@o3de.org](mailto:security@o3de.org) mailing list. Vulnerability reports should follow the same information reporting format as [standard O3DE issues](https://github.com/o3de/o3de/blob/development/.github/ISSUE_TEMPLATE/bug_template.md) along with any security specific details.
```

Add `mailto:` link to email address.

File: content/docs/contributing/security/_index.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+## When Should I Report a Vulnerability?
+* You have discovered a potential security issue relating to O3DE. 
+* You have discovered or are aware of a security issue relating to an O3DE dependency that may impact O3DE.
+     * If the issues is in a product that O3DE depends upon, we strongly encourage you to also engage with that project's vulnerability reporting mechanisms. 
+
+## When Should I NOT Report a Vulnerability?
+* You need help setting up or have issues relating to the use of security dependencies such as OpenSSL or TLS/DTLS
```
Comment (sptramer) [inline content/docs/contributing/security/_index.md:22]:
```suggestion
* You need help setting up or have issues relating to the use of security dependencies such as OpenSSL or TLS/DTLS.
```

File: content/docs/contributing/security/_index.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+## When Should I NOT Report a Vulnerability?
+* You need help setting up or have issues relating to the use of security dependencies such as OpenSSL or TLS/DTLS 
+* You need help applying a security related fix to your O3DE applications. 
+* The issue is not related to security. Please report issue using the normal [O3DE issue reporting process](https://github.com/o3de/o3de/issues).
+
+## Can I Report a Vulnerability via GitHub Issues?
+If there is publicly disclosed security information, such as a report in the [National Vulnerability Database](https://nvd.nist.gov/) then GitHub issues can be used to communicate security vulnerabilities impacting O3DE.
+
+# Security Vulnerability Response
+When a security vulnerability is reported via security@o3de.org, a member of the SIG-Security response team should respond within 3 working days. Once the issue has been acknowledged, the SIG-Security response team will follow its [documented response process](https://github.com/o3de/sig-security/issues/14) to ensure the reporter is updated and informed of current actions and next steps.
```
Comment (sptramer) [inline content/docs/contributing/security/_index.md:30]:
```suggestion
When a security vulnerability is reported via the [security mailing list](mailto:security@o3de.org), a member of the SIG-Security response team should respond within 3 working days. Once the issue has been acknowledged, the SIG-Security response team will follow its [documented response process](https://github.com/o3de/sig-security/issues/14) to ensure the reporter is updated and informed of current actions and next steps.
```

**Review by sptramer — CHANGES_REQUESTED** [review]
Minor copy changes to align with style but otherwise no comments or objections. Looks like standard vuln reporting.

File: content/docs/contributing/security/_index.md (hunk 6 above)
Comment (lmbr-pip) [reply content/docs/contributing/security/_index.md:9]: Fixed

File: content/docs/contributing/security/_index.md (hunk 7 above)
Comment (lmbr-pip) [reply content/docs/contributing/security/_index.md:14]: Fixed

**Comment by lmbr-pip** [comment]
Propose moving disclosure part to a different RFC, as and when needed.

File: content/docs/contributing/security/_index.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+toc: true
+weight: 100
+---
+
+## Open 3D Engine: Security
+This page describes the Security and Disclosure policies for **Open 3D Engine (O3DE)**.
+
+O3DE values all contributions that help keep O3DE secure. [SIG-Security](https://github.com/o3de/sig-security) volunteers will thoroughly research and investigate all reported vulnerabilities. 
+
+# Report a Vulnerability
```
Comment (sptramer) [inline content/docs/contributing/security/_index.md:13]:
```suggestion
# How to report a vulnerability

```

File: content/docs/contributing/security/_index.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+We encourage all vulnerabilities to be reported through the [security@o3de.org](mailto:security@o3de.org) mailing list. Vulnerability reports should follow the same information reporting format as [standard O3DE issues](https://github.com/o3de/o3de/blob/development/.github/ISSUE_TEMPLATE/bug_template.md) along with any security specific details.
+
+## When Should I Report a Vulnerability?
```
Comment (sptramer) [inline content/docs/contributing/security/_index.md:16]:
```suggestion
## When should I report a vulnerability?

```

File: content/docs/contributing/security/_index.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+* You have discovered a potential security issue relating to O3DE. 
+* You have discovered or are aware of a security issue relating to an O3DE dependency that may impact O3DE.
+     * If the issues is in a product that O3DE depends upon, we strongly encourage you to also engage with that project's vulnerability reporting mechanisms. 
+
+## When Should I NOT Report a Vulnerability?
```
Comment (sptramer) [inline content/docs/contributing/security/_index.md:21]:
```suggestion
## When should I NOT report a vulnerability?

```

File: content/docs/contributing/security/_index.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+* You need help setting up or have issues relating to the use of security dependencies such as OpenSSL or TLS/DTLS.
+* You need help applying a security related fix to your O3DE applications. 
+* The issue is not related to security. Please report issue using the normal [O3DE issue reporting process](https://github.com/o3de/o3de/issues).
+
+## Can I Report a Vulnerability via GitHub Issues?
```
Comment (sptramer) [inline content/docs/contributing/security/_index.md:26]:
```suggestion
## Can I report a vulnerability via GitHub Issues?

```

File: content/docs/contributing/security/_index.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+If there is publicly disclosed security information, such as a report in the [National Vulnerability Database](https://nvd.nist.gov/) then GitHub issues can be used to communicate security vulnerabilities impacting O3DE.
+
+# Security Vulnerability Response
```
Comment (sptramer) [inline content/docs/contributing/security/_index.md:29]:
```suggestion
# Security vulnerability response

```

File: content/docs/contributing/security/_index.md [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+When a security vulnerability is reported via the [security mailing list](mailto:security@o3de.org), a member of the SIG-Security response team should respond within 3 working days. Once the issue has been acknowledged, the SIG-Security response team will follow its [documented response process](https://github.com/o3de/sig-security/issues/14) to ensure the reporter is updated and informed of current actions and next steps.
+
+Any vulnerability information will be treated as confidential and will be not shared outside the security response process. Vulnerability information will not be publicly shared or disclosed without the consent of the reporter.
+
+# Public Disclosure
```
Comment (sptramer) [inline content/docs/contributing/security/_index.md:34]:
```suggestion
# Public disclosure policy

```

**Review by sptramer — CHANGES_REQUESTED** [review]
Small style guide and formatting changes.

File: content/docs/contributing/security/_index.md (hunk 14 above)
Comment (lmbr-pip) [reply content/docs/contributing/security/_index.md]: Fixed

File: content/docs/contributing/security/_index.md (hunk 12 above)
Comment (lmbr-pip) [reply content/docs/contributing/security/_index.md]: Fixed

File: content/docs/contributing/security/_index.md (hunk 11 above)
Comment (lmbr-pip) [reply content/docs/contributing/security/_index.md]: Fixed

File: content/docs/contributing/security/_index.md (hunk 10 above)
Comment (lmbr-pip) [reply content/docs/contributing/security/_index.md]: Fixed

**Review by sptramer — APPROVED** [review]

**Review by amzn-changml — APPROVED** [review]

**Pull request merged**